src/Scope.ts

```diff
@@ -1010,6 +1035,7 @@ interface NamespaceContainer {
     statements: Statement[];
     classStatements: Record<string, ClassStatement>;
     functionStatements: Record<string, FunctionStatement>;
+    enumStatements: Record<string, EnumStatement>;
```

Comment:
Let's use a Map<string, EnumStatement> here like we do for namespaces. (I intend on converting the class and function statements to using maps in the future

Reply:
done